smartsim/settings/arguments/launch/dragon.py

```diff
@@ -56,11 +59,16 @@ def set_tasks_per_node(self, tasks_per_node: int) -> None:
 
         :param tasks_per_node: number of tasks per node
         """
-        self.set("tasks-per-node", str(tasks_per_node))
+        self.set("tasks_per_node", str(tasks_per_node))
```

Comment:
Oh lol, this wasn't even wrong the first time! This was me being lazy so that I could use the `**` operator on `_launch_args` when formatting a `DragonRunRequestView`!